test/e2e/specs/editor/various/preferences.spec.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'preferences', () => {
+	test( 'remembers sidebar dismissal between sessions', async ( {
+		page,
+		admin,
+	} ) => {
+		await admin.createNewPost();
+		// Open by default.
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Post' );
+
+		// Change to "Block" tab.
+		await page.click( 'role=button[name="Block"i]' );
+
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Block' );
+
+		// Regression test: Reload resets to document tab.
+		//
+		// See: https://github.com/WordPress/gutenberg/issues/6377
+		// See: https://github.com/WordPress/gutenberg/pull/8995
+		await page.reload();
+		await page.waitForSelector( '.edit-post-layout' );
```

Comment:
This line essentially waits for the block editor to load, but I think it line can be removed. The line below it could be changed to `await expect.poll(`, which would make the test wait for the panel tab to be present (so it would have the same result of waiting for the editor to load).